src/style/app.module.css

```diff
@@ -30,7 +39,7 @@
   --dropdown-hover-color: #eff1f7;
   --grey-bg-color: #eaebef;
   --light-dark-green: #08780b;
-  --light-green: #31bb6b;
+  --grey-light: #31bb6b;
```

Comment:
1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466 
    1. https://www.figma.com/design/dmKt00m9GVSeA1nebnkxql/Untitled?node-id=0-1&node-type=canvas&t=TPzfh1DE9IxjzsEe-0
3. There are appropriate blue/grey colors that have been used in other screens
4. Using variables named --grey-light to reference a color that is clearly green is not in keeping with our design system in the figma and PR above. It is also not in keeping with the spirit of our community. 

![image](https://github.com/user-attachments/assets/6677fe89-cabf-42f4-8a89-0cf30bb996e4)